src/nvim/api/buffer.c

```diff
@@ -491,6 +491,43 @@ void nvim_buf_set_lines(uint64_t channel_id,
   try_end(err);
 }
 
+/// Return the byte offset for a line.
+//
+/// This includes the end-of-line character, depending on the 'fileformat'
+/// option for the current buffer.  The first line returns 0.
+/// UTF-8 encoding is used, 'fileencoding' is ignored.
+/// Sending in the index just below the last line gives
+/// the total byte count for the entire file. A final newline
+/// will be included if it would be written, see |eol|.
+///
+/// Unlike |line2byte|, throws error for out-of-bounds indexing.
```

Comment:
Technically the tag is `|line2byte()|`. `make html` will complain about this, I fixed several in add2a6239810a936f682d340525ac98e5e1fb15b.

For the same reason `|eol|` might need to be `|'eol'|` or just `'eol'`.